deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -760,18 +760,22 @@ spec:
                           clusterName:
                             type: string
                           host:
+                            description: Deprecated and to be removed on subsequent
+                              release. Use .URL with infinispan+xsite schema instead.
                             type: string
                           name:
                             type: string
                           namespace:
                             type: string
                           port:
+                            description: Deprecated and to be removed on subsequent
+                              release. Use .URL with infinispan+xsite schema instead.
                             format: int32
                             type: integer
                           secretName:
                             type: string
                           url:
-                            pattern: ^(kubernetes|minikube|openshift):\/\/(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])(:[0-9]+)?$
+                            pattern: (^(kubernetes|minikube|openshift):\/\/(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])*(:[0-9]+)+$)|(^(infinispan\+xsite):\/\/(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])*(:[0-9]+)*$)
```

Comment:
can't we replace the [a-z0-9] with `\w`?
the infinispan+xsite pattern should end with `(:[0-9]+)?` and not `(:[0-9]+)*`

Reply:
I've been playing around and I think it can be simplified like this
`(^(kubernetes|minikube|openshift|infinispan\+xsite):\/\/(\w|\w[\w\-\.]*\w)(:[0-9]+)?$)`
See https://regex101.com/r/YSCQe4/1

Reply:
It seems to be kubernetes://111 is valid, but this is not valid address
This complex regex validates correct IP address as well as hostname, also `kubernetes` must contains port, but `infinispan+xsite` not

Reply:
Got it from this library collection https://regexr.com/
It contains more and less complex solutions